features/step_definitions/git-flow.js

```diff
@@ -248,39 +101,65 @@ module.exports = function server() {
 
     this.When(/^it is targeting the pipeline's branch$/, () => null);
 
-    this.When(/^the pull request is closed$/, () => {
-        // Closing a PR requires authentication
-        github.authenticate({
-            type: 'oauth',
-            token: this.github_token
-        });
-
-        return github.pullRequests.update({
-            user: this.repositoryOwner,
-            repo: this.repository,
-            number: this.pullRequestNumber,
-            state: 'closed'
-        });
-    });
+    this.When(/^the pull request is closed$/, {
+        timeout: 60 * 1000
```

Comment:
Okay, got it. Seems like the timeout here is only for async call inside the before hook